cmd/syncer/main.go

```diff
@@ -47,6 +48,11 @@ var (
 	leaderElectionNamespace = flag.String("leader-election-namespace", "", "Namespace where the leader election resource lives. Defaults to the pod namespace if not set.")
 	printVersion            = flag.Bool("version", false, "Print syncer version and exit")
 	operationMode           = flag.String("operation-mode", operationModeMetaDataSync, "specify operation mode METADATA_SYNC or WEBHOOK_SERVER")
+
+	supervisorFSSName      = flag.String("supervisor-fss-name", "", "Name of the feature state switch configmap in supervisor cluster")
+	supervisorFSSNamespace = flag.String("supervisor-fss-namespace", "", "Location of the feature state switch configmap in supervisor cluster")
+	internalFSSName        = flag.String("fss-name", "", "Name of the feature state switch configmap")
+	internalFSSNamespace   = flag.String("fss-namespace", "", "Location of the feature state switch configmap")
```

Comment:
`Location of the feature` => `namespace of the feature state switch configmap`

Reply:
Done